### DISCUSSION TITLE
Lesson 7: ```forge test -m testPriceFeedVersionIsAccurate``` command throws error

### DISCUSSION THREAD

**Discussion by elec7rik** [Cyfrin/foundry-full-course-cu#776] [Q&A] [answered]
```forge test``` works fine but 
```
forge test -m testPriceFeedVersionIsAccurate
error: unexpected argument '-m' found
```
throws error. why could this be happening? i have imported ```{Test, console}, {FundMe}``` properly.

**Comment by Loscar1** [comment, marked as answer]
use --mt instead of -m

**Reply by elec7rik** [reply]
it worked, thanks. 
is this the context for -m & --mt? 
![mt](https://github.com/Cyfrin/foundry-full-course-f23/assets/89316185/7ae5f6d2-b4a2-49fe-bb97-ce098eb871f6)

**Reply by elec7rik** [reply]
yes it is, got it from the README.md. thanks.
